homeassistant/components/zha/core/gateway.py

```diff
@@ -454,6 +465,8 @@ def async_update_device(self, sender: zigpy_dev.Device, available: bool = True):
 
         device_info = zha_device.async_get_info()
 
+        self._hass.async_run_job(self._async_successfully_paired_feedback, zha_device)
```

Comment:
Why use `run_job` ? Instead put it after the `dispatcher_send` and just await it (or inline the function).

Reply:
fixed